skops/card/_model_card.py

```diff
@@ -19,6 +20,12 @@
 from skops.io import load
 from skops.utils.importutils import import_or_raise
 
+if sys.version_info >= (3, 11):
+    from typing import Self
+else:
+    from typing_extensions import Self
```

Comment:
is this another instance where mypy doesn't like it to be in `fixes.py`?

Reply:
Yes, it is indeed annoying.